src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperOutput.cs

```diff
@@ -36,9 +34,9 @@ public TagHelperOutput(string tagName, [NotNull] IDictionary<string, string> att
         {
             TagName = tagName;
             Attributes = new Dictionary<string, string>(attributes, StringComparer.OrdinalIgnoreCase);
-            PreContent = string.Empty;
-            _content = string.Empty;
-            PostContent = string.Empty;
+            PreContent = new DefaultTagHelperContent();
```

Comment:
think we decided not to bother with lazy initialization up front.  doubt `DefaultTagHelperContent` will show up in a profile.  if it does, the increase will likely be minuscule compared to the savings this PR should bring.  of course, need the before / after numbers and a profile after to confirm.